packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -612,6 +612,87 @@ class CustomScrollView extends ScrollView {
+  /// known as box widgets. (Widgets like [Container], [Row], and [SizedBox] are
+  /// box widgets.)
+  ///
+  /// While boxes are much more straightforward (implementing a simple
+  /// two-dimensional Cartesian layout system), slivers are much more powerful,
+  /// and are optimized for one-axis scrolling environments.
+  ///
+  /// Slivers are hosted in viewports, also known as scroll views, most notably
+  /// [CustomScrollView].
+  ///
+  /// ## Examples of slivers
+  ///
+  /// The Flutter framework has many built-in sliver widgets, and custom widgets
+  /// can be created in the same manner. By convention, sliver widgets always
+  /// start with the prefix `Sliver` and are always used in properties called
+  /// `sliver` or `slivers` (as opposed to `child` and `children` which are used
+  /// for box widgets).
+  ///
+  /// Examples of widgets unique to the sliver world include:
+  ///
+  /// * [SliverList], a lazily-loading list of variably-sized box widgets.
+  /// * [SliverFixedExtentList], a lazily-loading list of box widgets that are
+  ///   all forced to the same height.
+  /// * [SliverPrototypeExtentList], a lazily-loading list of box widgets that
+  ///   are all forced to the same height as a given prototype widget.
+  /// * [SliverGrid], a lazily-loading grid of box widgets.
+  /// * [SliverAnimatedList] and [SliverAnimatedGrid], animated variants of
+  ///   [SliverList] and [SliverGrid].
+  /// * [SliverFillRemaining], a widget that fills all remaining space in a
+  ///   scroll view, and lays a box widget out inside that space.
+  /// * [SliverFillViewport], a widget that lays a list of boxes out, each
+  ///   being sized to fit the whole viewport.
+  /// * [SliverPersistentHeader], a sliver that implements pinned and floating
+  ///   headers, e.g. used to implement [SliverAppBar].
+  /// * [SliverToBoxAdapter], a sliver that wraps a box widget.
+  ///
+  /// Examples of sliver variants of common box widgets include:
+  ///
+  /// * [SliverOpacity], [SliverAnimatedOpacity], and [SliverFadeTransition],
+  ///   sliver versions of [Opacity], [AnimatedOpacity], and [FadeTransition].
+  /// * [SliverIgnorePointer], a sliver version of [IgnorePointer].
+  /// * [SliverLayoutBuilder], a sliver version of [LayoutBuilder].
+  /// * [SliverOffstage], a sliver version of [Offstage].
+  /// * [SliverPadding], a sliver version of [Padding].
+  /// * [SliverReorderableList], a sliver version of [ReorderableList]
+  /// * [SliverSafeArea], a sliver version of [SafeArea].
+  /// * [SliverVisibility], a sliver version of [Visibility].
+  ///
+  /// ## Mixing slivers and boxes
+  ///
+  /// In general, slivers always wrap box widgets to actually render anything
+  /// (for example, there is no sliver equivalent of [Text] or [Container]);
+  /// the sliver part of the equation is mostly about how these boxes should
+  /// be laid out in a viewport (i.e. when scrolling).
```

Comment:
By the same argument, one could worry about people not understanding "part of the equation" or "mostly about" or "laid out" or indeed "for example". I prefer to assume that readers who are unsure about the meaning of the text will look up the meaning (which is trivial to do with today's web), respecting the audience's literacy, and potentially letting them learn something along the way. Certainly I personally find that reading texts that are rich with nuance and forms that are unfamiliar to me is a great way to expand my knowledge and improve my understanding of the world.

To put it another way, while it is important to ensure that our communication is clear and understandable, we should also avoid oversimplifying or limiting the vocabulary we use to communicate.

Reply:
OK, works for me.

I do think "unsure about the meaning of the text" isn't quite the issue (and I guess I euphemized it above at the expense of clarity): unlike the situation with lots of terms that may simply be unfamiliar, a lot of people are mistaken about the meaning of "i.e." and "e.g." and don't realize it, so they don't even know there's something they should look up.

You can tell because many people *write* "i.e." when in fact they're giving just a specific example. (Sometimes it's ambiguous whether someone's mistake is in the meaning of the term or in the facts of what they're talking about, but I see plenty of cases where it's harder to believe they'd be confused about the facts than about the term.)

Reply:
I mean, sure. People also misuse "their" and "they're" and "there" but I'm not going to stop using those either. :-)